wolfssl/internal.h

```diff
@@ -4283,10 +4292,13 @@ struct WOLFSSL {
     HandShakeDoneCb hsDoneCb;          /*  notify user handshake done */
     void*           hsDoneCtx;         /*  user handshake cb context  */
 #endif
+#ifdef WOLFSSL_ASYNC_IO
 #ifdef WOLFSSL_ASYNC_CRYPT
-    struct WOLFSSL_ASYNC async;
-#elif defined(WOLFSSL_NONBLOCK_OCSP)
-    void*           nonblockarg;        /* dynamic arg for handling non-block resume */
+    WC_ASYNC_DEV* asyncDev;
```

Comment:
Is `WC_ASYNC_DEV* asyncDev;` still needed/used?

Reply:
Yes, it is needed for async crypto. It also needs to be separate because it is not only used within the handshake machine context.